pandas/tests/test_internals.py

```diff
@@ -655,7 +655,7 @@ def test_interleave(self):
         mgr = create_mgr('a: f8; b: i8')
         self.assertEqual(mgr.as_matrix().dtype, 'f8')
         mgr = create_mgr('a: f4; b: i8')
-        self.assertEqual(mgr.as_matrix().dtype, 'f4')
+        self.assertEqual(mgr.as_matrix().dtype, 'f8')
```

Comment:
I can take another look at this.  It can go both ways.  There is loss of precision of big integers when going from `int64` -> `float32`.  But, in some scenarios, one just wants to keep things in `float32` for memory purposes (or if the rest of their data is `float32`, they might risk accidentally upcasting all of it it to `float64` if they get one `float64` in the mix)